akka-http/src/main/scala/scala/util/matching/JavaConverters.scala

```diff
@@ -0,0 +1,10 @@
+package scala.util.matching
```

Comment:
Comment did not explain it's package private :)
Sadly this will break osgi packaging and people will come at us with pitchforks and fire ;)
We should be able to achieve the same thing with some reflection, to avoid the package private problem them.